plugins/inputs/ipvs/ipvs.go

```diff
@@ -81,6 +110,17 @@ func serviceTags(s *ipvs.Service) map[string]string {
 	return ret
 }
 
+// helper: given a Destination, return tags that identify it
+func destinationTags(d *ipvs.Destination) map[string]string {
+	return map[string]string{
+		"address":        d.Address.String(),
+		"port":           fmt.Sprintf("%d", d.Port),
```

Comment:
I've been told `strconv.FormatInt` will do these conversions faster.

Reply:
Done.

Reply:
This one I still see as being an `Sprintf`